go.mod

```diff
@@ -1,9 +1,14 @@
 module github.com/sirupsen/logrus
 
+require (
+	github.com/stretchr/testify v1.10.0
+	golang.org/x/sys v0.13.0
+)
+
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/sys v0.0.0-20220715151400-c0bba94af5f8
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-go 1.13
+go 1.17
```

Comment:
nit: perhaps a good opportunity to move it to the top (at least I'm used to go version being at the start of go.mod) WDYT?

Reply:
done